Assets/_Game/Scripts/Visibility/VisibleGameObject.cs

```diff
@@ -40,7 +40,7 @@ protected virtual void Start()
         Show(CandleColor.Yellow);
     }
 
-    protected virtual void EventManager_onCandleColorChanged(CandleColor color)
+    protected virtual void EventManager_onCandleColorChanged(CandleColor color, float timeToLast)
```

Comment:
is this new param being used or going to be used?

Reply:
Looks like this method's parameters had to be updated to match the updated event it's listening to. Other classes are using the new param from the event.